### DISCUSSION TITLE
stuck to load and crash

### DISCUSSION THREAD

**Discussion by pecheh** [ppy/osu#22839] [Q&A]
When playing beatmaps with videos, there is a chance that the screen will freeze. I have previously posted about this issue on osu forum
1. [Getting Crash ](https://osu.ppy.sh/community/forums/topics/1140781?n=1)
2. [When beatmap have video, it may get stuck in loading](https://osu.ppy.sh/community/forums/topics/1592438?n=1)

It still hasn't been resolved. I created a GitHub account in the hope of getting more help.
The problem of the screen freezing has persisted for a long time. 

When the screen stuck to load:

1. In the case of solo games, pressing Alt+F4 can exit the game and return to the beatmap selection. 
However, if you continue playing without restart, the screen will become increasingly laggy .
Checking the task manager will show high CPU usage, and subsequently, the computer will become unresponsive to keyboard and mouse controls, with the screen frozen. The only solution is to force shut down the power switch and restart the computer. (in my pc , only osu have this problem )

2. In the case of multiplayer games, pressing Alt+F4 will log you out of the game. The subsequent situation is the same as in solo games.

Restarting the game can temporarily solve the problem, but it will occur again after a while. 

hope the problem can be solved , thanks.

### when problem occurs , high cpu usage
![when problem occurs , high cpu usage](https://user-images.githubusercontent.com/127624798/224508330-7eeff686-4eb1-4a79-bf52-89da7a06e807.jpg)
### stuck loading example
![screenshot506](https://user-images.githubusercontent.com/127624798/224508454-ebc42f16-9872-45be-864c-570f8f7571d5.jpg)

### osu!/Logs
[session.log](https://github.com/ppy/osu/files/10949311/session.log)
[update_success.log](https://github.com/ppy/osu/files/10949312/update_success.log)
[runtime.log](https://github.com/ppy/osu/files/10949314/runtime.log)

### When quit game in EventViewer
 [Eventviewer.txt](https://github.com/ppy/osu/files/10949320/Eventviewer.txt)
 
### Environment
CPU : Intel core i5 6400
RAM : 32 GB
GPU : Nvidia GTX 1050 ti
Win10 22H2 latest update
Nvidia Driver Version : 526.98
Resolution of screen : 1920x1080 (borderless)
Version of osu : Stable 20230206

**Comment by ItsShamed** [comment]
You should post this issue on https://github.com/ppy/osu-stable-issues, this is the repository for osu!lazer, the next version of osu!.

**Reply by pecheh** [reply]
oh , i'm sorry
i will post again on correct item
thank you very mush.
